FirebaseMessaging/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # 2021-04 -- v7.11.0
-- [changed] Refactor Messaging to internally not depending on InstanceID, but can co-exist. Will remove InstanceID dependency in the next Firebase breaking change. (#7814)
+- [changed] Refactor Messaging to internally not depend on InstanceID, but can co-exist. This makes Messaging's APIs of handling FCM registration token no longer depend on InstanceID to function. Recommend use Messaging to handle FCm registration tokens. (#7814)
```

Comment:
Consider: "Remove Messaging's dependency on InstanceID while continuing to optionally interoperate with it. This makes Messaging's APIs of handling FCM registration tokens no longer depend on InstanceID. We recommend migrating the handling of FCM registration tokens from InstanceID APIs to FCM APIs".

Reply:
Is there a link that demonstrates an example of migrating from IID to FCM?

Reply:
The doc is updated with Messaging API instead of IID since Fireconf: https://firebase.google.com/docs/cloud-messaging/ios/client#access_the_registration_token

I will add it in the comment.

Reply:
Done.